lib/posix/Kconfig

```diff
@@ -39,6 +39,36 @@ config SEM_VALUE_MAX
 	help
 	  Maximum semaphore count in POSIX compliant Application.
 
+config PTHREAD_DYNAMIC_STACK
+	bool "Support for dynamic stacks"
```

Comment:
Please spell it fully: "Support for dynamic thread stacks"

Reply:
To be honest, @pfalcon, I don't even think there should be a Kconfig for this. It's basically synonymous with "are we allowed to pass a NULL attr?" The spec explicitly says "yes" here. Maybe removing this would make it easier to solve the static / dynamic wording as well?

I guess the way you're leaning is that dynamic <=> heap-allocated, while static <=> statically allocated?

It would be nice to get @andrewboie 's input as well for naming.